src/inverseRobot.js

```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const val = Object.values(robot);
+
+  for (let counter = 0; counter <= val.length; counter++) {
+    let c = 1;
+
+    for (; c < val.length; c++) {
```

Comment:
you don't need two loops, you need to create a new empty object, iterate over the keys from the robot object and write the robot value as the key to the new object

Reply:
ok , I rewrite it)